CI/flatpak/com.obsproject.Studio.json

```diff
@@ -249,15 +249,15 @@
         {
           "type": "git",
           "url": "https://github.com/FFmpeg/FFmpeg.git",
-          "commit": "3980415627a187d188dc25669cea6b12912eb178",
+          "commit": "a6dc92968a325d331bb6dcf9b3b2248026cd1d6c",
           "disable-shallow-clone": true
         },
         {
           "type": "git",
           "dest": "obs-deps",
           "url": "https://github.com/obsproject/obs-deps.git",
-          "tag": "2023-03-30",
-          "commit": "1a977b8ac95ac2bab5e6dbfd34e029ad06933548"
+          "tag": "2023-04-03",
+          "commit": "55b333baa4c0f8a6ec6c5a7fdae34474bdfa9085"
```

Comment:
Note that you'll also have to update the command that applies our FFmpeg patches below since they're now split into two separate files.

Reply:
Ah. Will fix shortly.

Reply:
Force pushed. Should be fixed now.